core/scss/utilities/mixins/typography/_big-paragraph.scss

```diff
@@ -0,0 +1,12 @@
+@charset "UTF-8";
+
+//
+// @big-paragraph
+//
+// To be used for describing the content of an image.
+//
+// Style guide: Mixins.Typography.big-paragraph
+//
+@mixin big-paragraph {
+  font-size: 2rem;
```

Comment:
```suggestion
  font-size: 2rem;
  @include grid-media-max('sm') {
   font-size: 1.6rem;
  }
```

Reply:
Looks like big paragraph font-size is 16px in Figma for xs and sm.

Reply:
Thanks for adding the responsiveness for this!